magento/pwa-studio
# fix: sticky sidebar on checkout page no longer overlaps footer

## Description

Create Account sidebar was overlapping with footer.

## Related Issue





* Closes [PWA-784](https://jira.corp.magento.com/browse/PWA-784).
* Closes #2586.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Open dev tools, switch to ipad view, rotate to landscape.
2. Check out and get to order conf page.
3. Scroll down, ensure that sidebar/create account does not overlap with footer

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/e1ffcec11fa9449fc9f5714140a205a9.gif)](https://gyazo.com/e1ffcec11fa9449fc9f5714140a205a9)

## Checklist

* I have updated the documentation accordingly, if necessary.
